### ISSUE TITLE
Would be useful to have a simple interface for getting a single-time snapshot at a target population size

### ISSUE THREAD

**Issue by sclamons** [biocircuits/bioscrape#94]
**Use case**: I have a control circuit. I've already verified (or don't care about) how it behaves dynamically. I want to see what kind of stationary distribution it leads to, ideally with low overhead, so I only care about state once the lineage has reached some large number of cells.

**What I have to do currently**: 
* Guess how long it will take to get to the right number of cells.
* Simulate out to that time.
* If it doesn't finish in some reasonable amount of time, or if it eats up all of my free RAM, manually kill Python and turn down the target time until it finishes. 
* Check through every schnitz to see if it runs to the end of the simulation. If there are "enough" of those schnitzes, continue with analysis. If not, ramp up the target time until I get enough cells. This has to be done slowly, since population will grow (stochastically!) exponentially with time.
* Once I have enough cells, pull out all of the final states from each schnitz that's still alive at the end of the simulation into a new data structure and analyze from there. 

**What I would like to be able to do*:
* Set a target population size.
* Simulate out a lineage until the population reaches the target size, then return a dataframe or numpy array containing only final cell states. 

_Alternatively_, could set an approximate amount of real time to simulate for, something like "run this for fifteen minutes and give me what you have". 

Would this be practical? Could it run significantly faster/with less memory than a standard SimulateCellLineage?

**Comment by WilliamIX** [comment]
This is similar to simulating with a population controlled chemostat (something I very much want to implement) - not particularly hard, but would require a whole new lineage simulation loop.

**Comment by sclamons** [comment]
Yes! Chemostat would be nice. I've been "simulating" this by running a
bunch of independent single-cell lineages, which feels like I'm throwing
away half of my work and can't simulate evolution.

**Comment by WilliamIX** [comment]
I can walk you through how to implement this if you want to get inside a simulation loop - or I could just do it. It is pretty easy.
